edd-blockonomics.php

```diff
@@ -607,6 +610,48 @@ public function settings( $settings )
                       .__('<p>Click on <b>Get Started for Free</b> on <a href=\'https://www.blockonomics.co/merchants\' target=\'_blank\'>Blockonomics Merchants</a>. Complete the Wizard, Copy the API Key when shown here</p>', 'edd-blockonomics').'");
       }
 
+      var network_confirmations = $("select[name=\'edd_settings[edd_blockonomics_confirmations]\']").find(":selected").text();
```

Comment:
Such an elaborate warning is not required here. We can remove this.
Add change option  _2_ to _2 (recommended)_

Reply:
Message removed and option updated 3e6fe36